src/components/SearchError.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import { Typography } from '@mui/material';
+import { ShowData } from '../types';
+import { getMovieTrending } from '../helpers';
+import ShowCarousel from './ShowCarousel';
+
+interface ShowErrorProps {
+    /**
+     * String of query searched by user
+     */
+    query: string;
+}
+
+/**
+ * Rendered when MDB returns no results for a search.
+ *
+ * @returns {JSX.Element} | Search Error
+ */
+const SearchError: React.FC<ShowErrorProps> = ({ query }): JSX.Element => {
+    const [trending, setTrending] = useState<ShowData[] | null>(null);
+
+    useEffect(() => {
+        const handler = async () => {
+            const movies = await getMovieTrending();
+            setTrending(movies);
+        };
+        handler();
+    }, []);
```

Comment:
future: Might be nice to add a `useTrendingShows` to wrap this logic since I think this is duplicate from the home screen.

Reply:
#539